src/platform/src/common/features/create-organization/components/CreateOrganizationDialog.jsx

```
@@ -0,0 +1,844 @@
+    if (!fileValidation.isValid) {
+      setErrors((prev) => ({
+        ...prev,
+        logoFile: fileValidation.error,
+      }));
+      return;
+    }
+
+    // Clear file errors
+    if (errors.logoFile) {
+      const newErrors = { ...errors };
+      delete newErrors.logoFile;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)